evalml/tests/model_understanding_tests/test_permutation_importance.py

```diff
@@ -25,7 +25,6 @@
 
 class DoubleColumns(Transformer):
     """Custom transformer for testing permutation importance implementation.
-
```

Comment:
@freddyaboulton I tried to do something along the lines of https://github.com/alteryx/evalml/pull/3172/commits/f480a93a3b72c9d5ec7e78dbdba1fffba6ca8da0

I guess where it failed was that I was initializing the pipelines in the parameterization. I then tried to mark just the case with the target encoder as a noncore_dependency, rather than the whole test. I can go back and try to use a (component_graph, parameters) tuple instead--I chose to use pipeline classes because we also have a regression pipeline test / graph that we test, which might be hard to capture just using the component graph.

Reply:
Alas, I tried again, trying to keep the pipeline instances as parameters so that we don't have to determine which pipeline class to use to initialize the parameters and component graph. It doesn't work because our the noncore dependency Target Encoder--I'll just keep it as the class-style API :')